src/core/p5.Renderer.js

```diff
@@ -109,6 +109,94 @@ p5.Renderer.prototype.resize = function(w, h) {
   }
 };
 
+/**
```

Comment:
this is mostly a verbatim copy from p5.Renderer2D.get, except the `instanceof` check is gone, and the `readPixel` call is used instead.